review/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+# flake8: noqa
```

Comment:
is it needed here?

Reply:
> Just add __init__py under /vagrant dir remove the import from review/tests/**init**.py
> 
> It will work for you, I tested it 💯

After I added `/__init__py` to the root directory (_/vagrant_ dir) I got this error, even without changing other code
**PS C:\src\github.com\guybeckenstein\Pro4U> pipenv run pytest review/tests/ -v -rP
ImportError while loading conftest 'C:\src\github.com\guybeckenstein\Pro4U\conftest.py'.
ModuleNotFoundError: No module named 'Pro4U.conftest'**

Reply:
I guess that the interpreter is confused because there is a root `conftest` and another `conftest` nested in review App. I can rename `review/tests/conftest.py` to `review/tests/review_conftest.py` and this problem will be solved, without using `__init__py` files

Reply:
It is should work anyway
As mentioned, you should have empty __ini__.py in your app test directory,
Did you do that?